code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -132,9 +137,10 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/plumphelmet
 	seed = /obj/item/seeds/plump
 	name = "plump-helmet"
-	desc = "<I>Plumus Hellmus</I>: Plump, soft and s-so inviting~"
```

Comment:
I'd rather the description changed to something that people who haven't played df can understand, maybe that it can be safely eaten without problems unlike the majority of growable mushrooms